### PR TITLE
Fix some broken links

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: api/rest/v1/bulk_info_place.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ permalink: /api/rest/v1/bulk/info/place
 
 ## /v1/bulk/info/place
 
-Get basic information about multiple [places](/api/rest/v1/getting_started#place).
+Get basic information about multiple [places](/glossary.md#place).
```
Comment (juliawu) [inline api/rest/v1/bulk_info_place.md:15]: change all these to glossary.html instead of .md

Comment (shifucun) [reply api/rest/v1/bulk_info_place.md]: Done

**Review by juliawu — APPROVED** [review]
LGTM!

**Review by shifucun — COMMENTED** [review]
thanks!

**Pull request merged**